packages/odyssey-react-mui/src/OdysseyThemeProvider.tsx

```diff
@@ -13,8 +13,10 @@
 import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles";
 import { memo, ReactElement, useMemo } from "react";
 
-import { createTheme, deepmerge, ThemeOptions, DesignTokensOverride } from ".";
-import { createOdysseyMuiTheme } from "./theme";
+import { createTheme } from ".";
```

Comment:
For reasons I don't understand, this one import only works when it's from `.` instead of `@mui/material`

Reply:
Maybe it was a fluke? I just tested it, and it works in this file:
```ts
import { createTheme } from '@mui/material'
```

We even export this function from `@mui/material`.

Reply:
I'm still getting this same issue; no idea why!